src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  const keys = Object.keys(robot);
+
+  for (const key of keys) {
```

Comment:
```suggestion
  for (const [key, value] of Object.entries(robot)) {
```
By utilizing the `Object.entries()` method, you can conveniently access both the key and value simultaneously.